2363/3/views/header.slim

```diff
@@ -0,0 +1,5 @@
+  form action='post/delete_all' method='post'
+    div class="btn-group btn-group-toggle" data-toggle="buttons"
+      a role='button' class='btn btn-outline-dark' href='/' Main
+      a role='button' class='btn btn-outline-dark' href='add' Add link
+      button value='delete' name='_method' id='hidden' type='submit' class='btn btn-outline-dark' onclick="return confirm('Delete all?')" Delete all  
```

Comment:
Please setup [editorconfig](https://editorconfig.org/) for your editor and add a `.editorconfig` file to your folder so your editor will always add newline at the end of file.